debian/control

```diff
@@ -94,18 +96,18 @@ Section: devel
 Depends:
  gir1.2-flatpak-1.0,
  flatpak (= ${binary:Version}),
- ostree (>= 2017.6),
+ ostree (>= 2017.10),
  python3-gi,
  ${misc:Depends},
  ${python3:Depends},
  ${shlibs:Depends},
 Recommends:
  binutils,
+ bzr,
  elfutils,
  git,
  patch,
  unzip,
-Suggests: bzr
```

Comment:
This had been intentionally changed by us downstream so bzr didn't get pulled into the image - https://github.com/endlessm/flatpak/commit/0694288131e3cf269abc8305b5313eaff9e93518.

Reply:
I reverted this and force pushed

Reply:
OK

Reply:
Nitpick: It would be good to list this kind of deviation from the upstream Debian packaging in the `debian/changelog` in future, under the ‘resync to Debian packaging’ entry, to make it more obvious that it’s a diff we need to continue carrying.